<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TelaListagemClientes</class>
 <widget class="QMainWindow" name="TelaListagemClientes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ListagemClientes</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>1280</width>
     <height>720</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame_TopReqtangle">
      <property name="minimumSize">
       <size>
        <width>1280</width>
        <height>116</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #FFD580</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="pushButton_Sair">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>50</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-style: none;
font-size: 20px;
color: tomato;
background: #FFD580;
</string>
       </property>
       <property name="text">
        <string>Sair</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>1060</x>
         <y>50</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(255, 255, 255);
border-style: none;
border-radius: 5px;
font-size:20px;</string>
       </property>
       <property name="text">
        <string>Cadastrar</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_DownReqtangle">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #faf9f6</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_DadosCliente_1">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>1280</width>
           <height>80</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QPushButton" name="pushButton_EditarCliente">
          <property name="geometry">
           <rect>
            <x>1090</x>
            <y>29</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">border-style: none;
font-size: 20px;
color: #FFAA00;
background: #FAF9F6;
</string>
          </property>
          <property name="text">
           <string>Editar</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_TelefoneCliente">
          <property name="geometry">
           <rect>
            <x>760</x>
            <y>30</y>
            <width>161</width>
            <height>21</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font-size: 20px</string>
          </property>
          <property name="text">
           <string>(67) 99323-5472</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_EmailCliente">
          <property name="geometry">
           <rect>
            <x>410</x>
            <y>22</y>
            <width>181</width>
            <height>31</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font-size: 20px</string>
          </property>
          <property name="text">
           <string>cliente@gmail.com</string>
          </property>
         </widget>
         <widget class="QFrame" name="frame_Divider">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>65</y>
            <width>1280</width>
            <height>1</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>1280</width>
            <height>1</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>1</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border: 1px solid lightgray</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
         <widget class="QLabel" name="label_NomeCliente">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>30</y>
            <width>131</width>
            <height>16</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font-size: 20px</string>
          </property>
          <property name="text">
           <string>Nome Cliente</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_DeletarCliente">
          <property name="geometry">
           <rect>
            <x>1200</x>
            <y>30</y>
            <width>31</width>
            <height>31</height>
           </rect>
          </property>
          <property name="toolTipDuration">
           <number>-1</number>
          </property>
          <property name="styleSheet">
           <string notr="true">color: #FFAA00;
border-style: none;
background-repeat: no-repeat;
background-image: url(:/Icons/Assets/Icons/trashIcon.png);
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>601</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="file_rc.qrc"/>
 </resources>
 <connections/>
</ui>
